src/components/common/promotioncard.tsx

```diff
@@ -35,14 +43,21 @@ const PromotionCard: FC<Props> = ({ promotion }) => {
 const InfoField = ({
   title,
   data,
+  className,
 }: {
   title: string;
   data: string | number;
+  className: string;
```

Comment:
Make className optional or pass default value in line 46 and remove the empty string in line 31.

Reply:
Done